staged_dependencies.yaml

```diff
@@ -25,6 +25,9 @@ upstream_repos:
   insightsengineering/rtables:
     repo: insightsengineering/rtables
     host: https://github.com
+  insightsengineering/rtables.officer:
+    repo: insightsengineering/rtables.officer
+    host: https://github.com
```

Comment:
i thought you guys have stopped using staged.dependnecies?

Reply:
That's correct, we're in the middle of migrating ([so far only teal is migrated](https://github.com/insightsengineering/nestdevs-tasks/issues/65)), But, we're working on other tasks so it is on hold until we get to it.